src/Glpi/Form/Destination/CommonITILField/AssociatedItemsFieldStrategy.php

```diff
@@ -132,62 +145,82 @@ private function getAssociatedItemsForSpecificAnswer(
             return null;
         }
 
-        $value = $answer->getRawAnswer();
-        if (!$this->isValidAnswer($value)) {
+        $values = $answer->getRawAnswer();
+        if (count($values) == count($values, COUNT_RECURSIVE)) {
+            $values = [$values];
+        }
```

Comment:
I guess the purpose of it is to transform a single item (`['itemtype' => 'xxx', 'items_id' => 123]`) into a list of items `[ ['itemtype' => 'xxx', 'items_id' => 123] ]`. If you want to check whether the arry is a list, you can use `if (!array_is_list($values)) { $values= [$values]; }`.

Also, a code comment would be appreciated.

Same remark for other occurences.